README.md

```diff
@@ -2,39 +2,32 @@
-- **Office connected to a Microsoft 365 subscription.** You might qualify for a Microsoft 365 E5 developer subscription through the [Microsoft 365 Developer Program](
-https://developer.microsoft.com/microsoft-365/dev-program), see [FAQ](
-https://learn.microsoft.com/office/developer-program/microsoft-365-developer-program-faq#who-qualifies-for-a-microsoft-365-e5-developer-subscription-) for details.
-Alternatively, you can [sign up for a 1-month free trial](
-https://www.microsoft.com/microsoft-365/try?rtc=1)
-or [purchase a Microsoft 365 plan](
-https://www.microsoft.com/microsoft-365/buy/compare-all-microsoft-365-products).
-
+- Node.js (the latest LTS version). Visit the [Node.js site](https://nodejs.org/) to download and install the right version for your operating system. To verify if you've already installed these tools, run the commands `node -v` and `npm -v` in your terminal.
+- Office connected to a Microsoft 365 subscription. You might qualify for a Microsoft 365 E5 developer subscription through the [Microsoft 365 Developer Program](https://developer.microsoft.com/microsoft-365/dev-program), see [FAQ](https://learn.microsoft.com/office/developer-program/microsoft-365-developer-program-faq#who-qualifies-for-a-microsoft-365-e5-developer-subscription-) for details. Alternatively, you can [sign up for a 1-month free trial](https://www.microsoft.com/microsoft-365/try?rtc=1) or [purchase a Microsoft 365 plan](https://www.microsoft.com/microsoft-365/buy/compare-all-microsoft-365-products).
 
 ### Create your add-in
 Use the Office Add-ins Development Kit for Visual Studio Code to set up your first add-in project. Create your Office add-in project using the following steps:
 
-* Ensure you've installed the `Microsoft Office Add-ins Development Kit for Visual Studio Code`.
-* Select the `Office Add-ins Development Kit` icon in the Visual Studio Code sidebar.
-* Select `Create a New Add-in` button from the sidebar.
-* Select an Office app that you want to build the add-in for.
+* Ensure you've installed the Microsoft Office Add-ins Development Kit for Visual Studio Code.
+* Select the Office Add-ins Development Kit icon in the Visual Studio Code **Activity Bar** to open the extension.
+* Select **Create a New Add-in**.
+* In the drop down that opens, select an Office app that you want to build the add-in for.
 * Select an add-in template from the list of available templates.
 * Select JavaScript/TypeScript as the programming language.
-* Choose a location where your new add-in will be created in a new folder.
-* Type a name for your project and hit Enter.
+* In the **Workspace folder** dialog that opens, select the folder where you want to create the project.
+* Give a name to the project (with no spaces) when prompted. Office Add-ins Development Kit will create the project with basic files and scaffolding. It will then open the project *in a second Visual Studio Code window*. Close the original Visual Studio Code window.

```

Comment:
I think it'll emphasize the behavior better if you italicize less of the line.
```suggestion
* Give a name to the project (with no spaces) when prompted. Office Add-ins Development Kit will create the project with basic files and scaffolding. It will then open the project in a *new* Visual Studio Code window. You are free to close the original Visual Studio Code window.

```